emily/cdk/lib/emily-stack.ts

```diff
@@ -37,12 +37,14 @@ export class EmilyStack extends cdk.Stack {
             ? cdk.RemovalPolicy.DESTROY
             : cdk.RemovalPolicy.RETAIN;
 
+        const enablePointInTime: undefined | boolean = EmilyStackUtils.isDevelopmentStack() ? undefined : true;
```

Comment:
Okay, then consider this resolved, but can you add a small comment above so that someone doesn't undo this some time in the future without knowing? Because this is unusual, but it's for a good reason

Reply:
Sure!